solutions/platinum/cses-1160.mdx

```diff
@@ -199,4 +199,88 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class PlanetQueries2 {
+	static Kattio x = new Kattio();
+	static PrintWriter pw = new PrintWriter(System.out);
+	static int n;
+	static int q;
+	static int[] arr;
+	static int[][] binaryLifting;
+	static int[] len;
```

Comment:
alot of these variables don't have to be global

Reply:
since all the helper functions, are absorbed inside the main, these don't need to be global anymore. removing it.